src/Panel/Panel.php

```diff
@@ -89,14 +516,80 @@ public static function link(App $kirby): bool
         return true;
     }
 
+    /**
+     * Filters the data array and returns only
+     * entries requested via a Fiber partial
+     * request (or the whole array if full request).
+     *
+     * @param \Kirby\Cms\App $kirby
+     * @param string $component
+     * @param array $data
+     * @return array
+     */
+    public static function partial(App $kirby, string $component, array $data): array
+    {
+        // is it a partial request?
+        $request          = $kirby->request();
+        $include          = $request->header('X-Fiber-Include')   ?? get('_include');
+        $requestComponent = $request->header('X-Fiber-Component') ?? get('_component');
+
+        // split include string into an array of fields
+        $include = Str::split($include);
```

Comment:
For a better overview and to avoid future regressions:

```suggestion
        $include = Str::split($include, ',');
```